R/comparePlot.R

```diff
@@ -142,72 +135,66 @@ comparePlot <- function(compareDF,
+    }
+
+    if (is.null(ylab)) {
+        ylab <- colnames(compareDF)[2]
+    }
     # Now make the column names suitable for use with aes_string
-    x = make.names(colnames(compareDF)[1])
-    y = make.names(colnames(compareDF)[2])
-    colnames(compareDF)[1:2] = make.names(colnames(compareDF)[1:2])
-
-    # SIMPLE PLOT: plot all data (no significance measures supplied)
-    if (is.null(compareDF[["xp"]]) | is.null(compareDF[["yp"]])) {
-        CompPlot = ggplot(compareDF, aes_string(x = x, y = y)) +
-            geom_point(shape = 1,
-                       size = symbolSize[["xUnique"]],
-                       color = symbolFill[["xUnique"]],
-                       fill = symbolFill[["xUnique"]],
-                       alpha = alpha) +
-            coord_equal(xlim = c(-scalemax, scalemax), ylim = c(-scalemax, scalemax))
-        if (dens2D == TRUE) {
-            CompPlot = CompPlot + geom_density2d(color = symbolFill[["yUnique"]])
-        }
-    } else
-        # DELUXE PLOT: plot groups in different colors/shapes
-        if (!is.null(compareDF[["xp"]]) & !is.null(compareDF[["yp"]])) {
-            # Plot the subsets
-            xindx = compareDF[["xp"]] <= pThreshold
-            yindx = compareDF[["yp"]] <= pThreshold
-
-            # Boolean indexes to parse groups
-            bothindx = xindx & yindx
-            neitherindx = !xindx & !yindx
-            xindx = xindx & !bothindx # Unique to X
-            yindx = yindx & !bothindx # Unique to y
-
-            # Create group factor column in compareDF
-            compareDF$group = NA
-            compareDF$group[bothindx] = "Common"
-            compareDF$group[xindx] = "X Unique"
-            compareDF$group[yindx] = "Y Unique"
-            compareDF$group[neitherindx] = "Not Significant"
-            compareDF <- compareDF %>%
-                dplyr::left_join(ssc)
-            compareDF$group <- compareDF$group %>%
-                factor(levels = c("Common", "X Unique", "Y Unique", "Not Significant"))
-
-            # Set an order field to control order of plotting
-            # Plot order is 1) Not Significant plotted first, 2) randomly plot X and Y Unique
-            # then plot Common last
-            compareDF$order = sample.int(nrow(compareDF)) + # Add random order (seeded for reproducibility)
-                nrow(compareDF) * (compareDF$group == "Common") +  # Assign common a high value to sort last
-                -nrow(compareDF) * (compareDF$group == "Not Significant") # Assign NotSig group neg values to sort first
+    x <- make.names(colnames(compareDF)[1])
+    y <- make.names(colnames(compareDF)[2])
+    colnames(compareDF)[1:2] <- make.names(colnames(compareDF)[1:2])
+
+    if (!is.null(compareDF[["xp"]]) & !is.null(compareDF[["yp"]])) {
+        sigMeasurePlot <- TRUE
+        #create group factor column in compareDF
+        compareDF$group                                                                        <- "Not Significant"
```

Comment:
don't line these up this far over to the right please

Reply:
ok, I did this to make it aligned with other group assignments. I will fix it